homeassistant/components/zwave_js/cover.py

```diff
@@ -140,18 +141,24 @@ def current_cover_position(self) -> int | None:
     async def async_set_cover_position(self, **kwargs: Any) -> None:
         """Move the cover to a specific position."""
         target_value = self.get_zwave_value(TARGET_VALUE_PROPERTY)
+        if target_value is None:
```

Comment:
I'm not sure that this can actually happen, but I think we can look at this in more detail later. I didn't see from the discovery schemas that we guard for it, but there might be some zwave standards that guard for it.

Reply:
we have a lot of checks like this. We should probably come up with a better plan

Reply:
Yeah, we should look over this in the whole integration. :+1: